agents/future-roadmap.md

```
@@ -107,25 +105,42 @@ RD-023 (Rule diagnostics)    — depends on RD-007; enhanced by RD-021 if availa
-- `date` field is `string | RecurConfig` — string = one-time date, RecurConfig = recurring
+- `date` field is `string | RecurConfig` — string = one-time date, `RecurConfig` = recurring
+
+**API model (from Swagger + live payload):**
+- **Response `Schedule`** returns: `id`, `name`, `rule`, `next_date`, `completed`, `posts_transaction`, `payee`, `account`, `amount`, `amountOp`, `date`
+- **Input `ScheduleInput`** accepts: `name?`, `posts_transaction?`, `payee?`, `account?`, `amount?`, `amountOp?`, `date` (**required**)
+- `rule`, `next_date`, and `completed` are **read-only server fields** for this page; do not send them in create / update payloads
+- `date` is either a one-time ISO date string or a full `RecurConfig` object; never send both representations together
+- `amount` is either a number or `{ num1, num2 }` when `amountOp === "isbetween"`
 
 **API endpoints:**
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)